CaptureSDK/GiniCaptureSDK/Sources/GiniCaptureSDK/Resources/GiniColors.swift

```diff
@@ -1,76 +1,151 @@
-        static let error4 = UIColorPreferred(named: "Error04")
-        static let error5 = UIColorPreferred(named: "Error05")
-
-        static let light1 = UIColorPreferred(named: "Light01")
-        static let light2 = UIColorPreferred(named: "Light02")
-        static let light3 = UIColorPreferred(named: "Light03")
-        static let light4 = UIColorPreferred(named: "Light04")
-        static let light5 = UIColorPreferred(named: "Light05")
-        static let light6 = UIColorPreferred(named: "Light06")
-
-        static let success1 = UIColorPreferred(named: "Success01")
-        static let success2 = UIColorPreferred(named: "Success02")
-        static let success3 = UIColorPreferred(named: "Success03")
-        static let success4 = UIColorPreferred(named: "Success04")
-        static let success5 = UIColorPreferred(named: "Success05")
-
-        static let warning1 = UIColorPreferred(named: "Warning01")
-        static let warning2 = UIColorPreferred(named: "Warning02")
-        static let warning3 = UIColorPreferred(named: "Warning03")
-        static let warning4 = UIColorPreferred(named: "Warning04")
-        static let warning5 = UIColorPreferred(named: "Warning05")
-
-        // GiniColors OLD
-        static let systemWhite = UIColorPreferred(named: "systemWhite")
-        static let label = UIColorPreferred(named: "label")
-        static let labelWhite = UIColorPreferred(named: "labelWhite")
-        static let grayLabel = UIColorPreferred(named: "grayLabel")
-        static let errorBackground = UIColorPreferred(named: "errorBackground")
-        static let helpBackground = UIColorPreferred(named: "helpBackground")
-        static let separator = UIColorPreferred(named: "separator")
-        static let subheadline = UIColorPreferred(named: "subheadline")
-        static let systemBlue = UIColorPreferred(named: "systemBlue")
-        static let systemGray = UIColorPreferred(named: "systemGray")
-        static let systemGray02 = UIColorPreferred(named: "systemGray02")
-        static let systemGray03 = UIColorPreferred(named: "systemGray03")
-        static let systemGray04 = UIColorPreferred(named: "systemGray04")
-        static let systemGray05 = UIColorPreferred(named: "systemGray05")
-        static let systemGray06 = UIColorPreferred(named: "systemGray06")
-        static let systemGreen = UIColorPreferred(named: "systemGreen")
-        static let systemIndigo = UIColorPreferred(named: "systemIndigo")
-        static let systemOrange = UIColorPreferred(named: "systemOrange")
-        static let systemPink = UIColorPreferred(named: "systemPink")
-        static let systemPurple = UIColorPreferred(named: "systemPurple")
-        static let systemRed = UIColorPreferred(named: "systemRed")
-        static let systemTeal = UIColorPreferred(named: "systemTeal")
-        static let systemYellow = UIColorPreferred(named: "systemYellow")
+        // Convert enum case to UIColor
+        public var toUIColor: UIColor {
```

Comment:
check if it should stay `public`

Reply:
I will remove it since is not needed. Thanks for catching this!